app/metrics.yaml

```diff
@@ -23,3 +23,100 @@ distribution:
     notification_emails:
       - dmu@mozilla.com
     expires: "2020-05-01"
+url:
+  domains:
+    type: counter
+    lifetime: ping
+    send_in_pings:
+      - session_end
+    description: >
+      Counting how many domains are visited in a session.
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+  visits:
+    type: counter
+    lifetime: ping
+    send_in_pings:
+      - session_end
+    description: >
+      Counting how many URL links are visited in a session.
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+  query_type:
+    type: labeled_counter
+    lifetime: ping
+    send_in_pings:
+      - session_end
+    description: >
+      Counting how many URLs are visited in a session, by input method.
+    labels:
+      - type_link
+      - type_query
+      - voice_query
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+controls:
```

Comment:
From this point on all metrics are missing the `send_in_pings` property and won't be sent in your newly defined ping.

Reply:
Those are discussing with my team. They haven't decided `tray_new_window` and `context_menu_long_pressed` counters if they should send at the end of a session and can get a reset in the new session. Or,  they wanna count them daily, so they would be ping lifetime.

`curve_mode_actived` is a boolean, so it can  be sent at its ping lifetime for sure.

Reply:
> 
> 
> Those are discussing with my team. They haven't decided `tray_new_window` and `context_menu_long_pressed` counters if they should send at the end of a session and can get a reset in the new session. Or, they wanna count them daily, so they would be ping lifetime.
> 
> `curve_mode_actived` is a boolean, so it can be sent at its ping lifetime for sure.

Please note that the fact that a metric is `lifetime: ping` has nothing to do with session: if you add the metric to your custom ping being send in `onStop`, then metrics will be cleared right after. If you add them to the `metrics` ping, then they will be cleared when that ping is sent, once per day. So it really depends on the ping.

I would recommend to keep things consistent, otherwise you'd need to two different analyses types.

Reply:
After the discussion with our team, we are going to remove these UI control telemetry in the following version, so we don't need to support them.